From 18ce91aae2e9efb4bcdc53008ea9a6e2e2a9b7f4 Mon Sep 17 00:00:00 2001
From: Tejasree Kondoj <ktejasree@marvell.com>
Date: Tue, 9 May 2023 11:55:32 +0530
Subject: [PATCH 414/955] test/crypto: fix IPsec AES-CCM test vector

Fixing IPsec AES-CCM test vector IV length.

Fixes: d314299950de ("test/crypto: add AES-CCM vectors")

Signed-off-by: Tejasree Kondoj <ktejasree@marvell.com>
Change-Id: I24af8fc62d879385e4d1d20e276ee8e7ea3f3ff9
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/103407
Base-Builds: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Base-Tests: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Reviewed-by: Anoob Joseph <anoobj@marvell.com>
---
 app/test/test_cryptodev_security_ipsec_test_vectors.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/test/test_cryptodev_security_ipsec_test_vectors.h b/app/test/test_cryptodev_security_ipsec_test_vectors.h
index 2686bbeb62afc..462cef6785bb0 100644
--- a/app/test/test_cryptodev_security_ipsec_test_vectors.h
+++ b/app/test/test_cryptodev_security_ipsec_test_vectors.h
@@ -417,7 +417,7 @@ struct ipsec_test_data pkt_aes_256_ccm = {
 				.op = RTE_CRYPTO_AEAD_OP_ENCRYPT,
 				.algo = RTE_CRYPTO_AEAD_AES_CCM,
 				.key.length = 32,
-				.iv.length = 12,
+				.iv.length = 11,
 				.iv.offset = IV_OFFSET,
 				.digest_length = 16,
 				.aad_length = 12,
-- 
2.25.1

